RN for PMM 2.36.0 (#1011)
percona/pmm-doc: docs/release-notes/2.36.0.md

- An indication of buckets as "done" vs "current" eliminates the need for the tool to evaluate the bucket status and facilitates the accurate data display
- The generation of a unique ID of a query, independent from PostgreSQL versions, database, users or schemas, improves observability of query planning, execution and performance

For more information on the enhancements included in PGSM2, please refer to the [full list] (https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md)available.

### Experimental Kubernetes Dashboards

We are thrilled to introduce three new dashboards in PMM V2.36, located in the Experimental folder of the Dashboards section. These dashboards, named Kubernetes Cluster Summary, Kubernetes Pods Status, and Kubernetes Volumes, provide valuable insights into Kubernetes environments. We encourage you to test these experimental dashboards and share your feedback with us on the forum, helping us improve them for future releases.

Add more subsections for more release highlights.

> 

Reviewer: ```suggestion
###New Advisors presentation

Understanding what advisors you have wasn't simple in PMM. from 2.36, we are presenting checks we have in advisors grouped by Categories and Advisors.
 
###Advisors for PostgreSQL executed against all DB.

From this release, all Advisors related to PostgreSQL will be executed against all available Databases, not only the one connected to PMM. This will provide you with insights about all DBs in your PostgreSQL server. 

###Backup management is enabled by default.

Backup management in PMM is enabled by default from this release, so you only need to configure your DBs to use this feature. LNK_TO_PMM_DOC
```

Reviewer: @catalinaadam ^ PTAL , the initial list

- Support for PostgreSQL 15 
- Enhanced consistency with `pg_stat_statements` so that the `pg_stat_monitor` view has identical column names, columns, and data types as `pg_stat_statements` for every major version of PostgreSQL from versions 11 to 15.
- A bucket status indication (done vs. current) eliminates the need for the tool to evaluate bucket status and facilitates accurate data display.
- The generation of a unique ID for a query makes it easier to monitor query planning, execution, and performance regardless of version, database, user, or schema.

For more information on the enhancements in PGSM2, refer to the [full list](https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md).

### Revamped Advisors page

We’ve improved the structure of the **Advisors** page to group Advisors by categories and clearly show the Advisors available for your current subscription. 

With this change, we have renamed all the tabs on the page, and added an option to easily integrate with Percona Platform for extra Advisors for free:

  ![!image](../_images/New_Advisors_page.PNG)

### PostgreSQL Advisors executed against all DBs

> Starting with this release, PostgreSQL Advisors are now executed against all the available databases (DBs), not only the one connected to PMM. This will provide you with insights about all databases in your PostgreSQL server.

Reviewer 2: Some PG checks need only default DB to perform needed operation, some need to fetch data from all DBs in the PG instance.
```suggestion
Starting with this release, PostgreSQL Advisors can check all available databases (DBs), not only the one connected to PMM. This will provide you with insights about all databases in your PostgreSQL server.
```

### pg_stat_monitor 2.0 support in QAN

We are excited to announce PMM 2.36 now supports [pg_stat_monitor](https://docs.percona.com/pg-stat-monitor/index.html) 2.0 (PGSM 2.0) in QAN, a powerful PostgreSQL query performance monitoring tool. By downloading this update, you will have access to the latest improvements and fixes covered by PGSM2, including:

- Improved internal architecture that results in fewer lock acquisitions and increases performance by approximately 20%.
- Support for PostgreSQL 15 
- Enhanced consistency with `pg_stat_statements` so that the `pg_stat_monitor` view has identical column names, columns, and data types as `pg_stat_statements` for every major version of PostgreSQL from versions 11 to 15.
- A bucket status indication (done vs. current) eliminates the need for the tool to evaluate bucket status and facilitates accurate data display.
- The generation of a unique ID for a query makes it easier to monitor query planning, execution, and performance regardless of version, database, user, or schema.

For more information on the enhancements in PGSM2, refer to the [full list](https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md).

> 

Reviewer 3: - PG versions 11 and 12 together with PGSM 2.0 will be supported in PMM 2.37

- [PMM-11363](https://jira.percona.com/browse/PMM-11363) - Starting with PMM 2.36.0, you no longer need to unassign a role before deleting it. If you try to delete a role assigned to a user, you will see a dropdown with replacement roles. As a result, the selected role will be assigned to the user.

## Improvements

- [PMM-11182](https://jira.percona.com/browse/PMM-11182) - Renamed the **Disconnect** button on the confirmation message that pop-ups when you are terminating Percona Platform connections as an Administrator. The button is now called **Force disconnect** to better differentiate between situations where you are disconnecting using a Percona Platform Account.

    Force-disconnecting only clears the connection from PMM without automatically removing the disconnected servers from Percona Platform as well. We recommend that you always disconnect while logged in with your Percona Account. Otherwise, make sure to manually remove the forced-disconnected servers from your [list of PMM instances in Percona Platform](https://portal.percona.com/pmm-instances).	

> - [PMM-11381](https://jira.percona.com/browse/PMM-11381) - Starting with PMM 2.34.0, you can scrape Kubelet metrics with DBaaS or VictoriaMetrics operator. Using Kubelet metrics, you can gather volume-related information.

Reviewer 4: Starting with PMM 2.36.0

Author: Fixed.

- [PMM-11566](https://jira.percona.com/browse/PMM-11566) - You now have Admin Access for PMM Demo instances so that you can experiment with all the features of PMM.

- [PMM-11699](https://jira.percona.com/browse/PMM-11699) - Improve documentation and descriptions for Advisor checks.
- [PMM-10974](https://jira.percona.com/browse/PMM-10974) - Usage of pg_stat_monitor by default

> - [PMM-11267](https://jira.percona.com/browse/PMM-11182) - Percona Server Enterprise features - Plugins enabled

Author: @catalinaadam Could you please check this?

## Bugs Fixed

> - [PMM-8146](https://jira.percona.com/browse/PMM-8146) - [DBaaS]: A user could see services, agents, and nodes associated with DB clusters that had already been deleted in DBaaS. The issue has been resolved now.

Reviewer 4: PMM-8146 is still in To Do

Author: Removed

> - [PMM-8251](https://jira.percona.com/browse/PMM-8251) - [DBaaS]: Despite the cluster being forcefully unregistered from DBaaS, metrics were still collected. Unregistering K8s should prevent metrics from being collected. This issue has been resolved now.

Reviewer 4: Same with PMM-8251 - in To Do

Author: Removed.

- [PMM-11291](https://jira.percona.com/browse/PMM-11291) - [DBaaS]: There was a difference in the shape and color of the DB Cluster Status box compared to the Kubernetes Cluster status box. The issue has been resolved now.
- [PMM-11296](https://jira.percona.com/browse/PMM-11296) - [DBaaS]: Fixed an issue where the DB Cluster creation page did not load when DBaaS was disabled.
- [PMM-11358](https://jira.percona.com/browse/PMM-11358) - [DBaaS]: Connection to the DB cluster displayed a loader when the DB cluster was suspended/paused. The issue has been resolved now.
- [PMM-11613](https://jira.percona.com/browse/PMM-11613) - [DBaaS]: Backups could not be created when uppercase letters were used in the storage name. The issue has been resolved now.

> - [PMM-11568](https://jira.percona.com/browse/PMM-11568) - [DBaaS]: Disabled the selection of MySQL versions less than 8.0.27.

Reviewer 4: PMM-11568 also not fixed

Author: Removed

<!---

> ### pg_stat_monitor 2.0 support in QAN

Reviewer 5: @capri1989 @JiriCtvrtka could you please review this part? Thanks

Reviewer 6: LGTM

- Its improved internal architecture leads to fewer lock acquisitions, and therefore an improved performance by approximately 20% when tested using pgbench.
- Support for PostgreSQL 15 
- Improved consistency with pg_stat_statements - Now pg_stat_monitor view has exactly the same column names, columns and data types as the pg_stat_statements view for every major version of PostgreSQL from 11 to 15.
- An indication of buckets as "done" vs "current" eliminates the need for the tool to evaluate the bucket status and facilitates the accurate data display
- The generation of a unique ID of a query, independent from PostgreSQL versions, database, users or schemas, improves observability of query planning, execution and performance

For more information on the enhancements included in PGSM2, please refer to the [full list] (https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md)available.

> ### Experimental Kubernetes Dashboards

Reviewer 5: @cshiv @anton-bystrov please feel free to review this part
